node/CHANGELOG.md

```diff
@@ -80,8 +83,6 @@ All notable changes to this project will be documented in this file.  The format
 * Limit the maximum number of clients connected to the event stream server via the `[event_stream_server][max_concurrent_subscribers]` config option.
 * Avoid emitting duplicate events in the event stream.
 * Change `BlockIdentifier` params in the Open-RPC schema to be optional.
-* Asymmetric connections are now swept regularly again.
```

Comment:
What happened here?

Reply:
These were fixed but not in 1.3.0 (it's broken there).

Looks like I should move this up to the `Unreleased` section.

Reply:
Fixed in d6c33b569